src/routes/cache-hooks/cache-hooks.service.ts

```diff
@@ -275,14 +287,62 @@ export class CacheHooksService {
             safeAddress: event.address,
           }),
         );
+        this._logMessageEvent(event);
         break;
       case EventType.CHAIN_UPDATE:
         promises.push(this.chainsRepository.clearChain(event.chainId));
+        this._logEvent(event);
         break;
       case EventType.SAFE_APPS_UPDATE:
         promises.push(this.safeAppsRepository.clearSafeApps(event.chainId));
+        this._logEvent(event);
         break;
     }
     return Promise.all(promises);
   }
+
+  private _logSafeTxEvent(
+    event: ExecutedTransaction | NewConfirmation | PendingTransaction,
+  ): void {
+    this.loggingService.info({
```

Comment:
I would add a "type" for this log for grouping and filtering these logs.

Reply:
Added in [d628fae](https://github.com/safe-global/safe-client-gateway/pull/852/commits/d628fae78be188e82ba36dc8ad022fabe899e41b)